doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-workflow.rst

```diff
@@ -86,10 +85,6 @@ teuthology. This procedure explains how to run tests using teuthology.
         -e <email>    When tests finish or time out, send an email to the
                       specified address. Can also be specified in 
                       ~/.teuthology.yaml as 'results_email'
-        -R            A comma-separated list of statuses to be used
-                      with --rerun. Supported statuses: 'dead',
-                      'fail', 'pass', 'queued', 'running', 'waiting'
-                      [default: fail,dead]
```

Comment:
This is good to remove here but how about expanding the last section in this doc about what `-R` does (including what the default is)?

Reply:
Done.